### PR TITLE
Add missing commonJS and AMD wrappers

### LINKED ISSUE #72: 1.3.0 doesn't include support for CommonJS, etc...
It was included in the framework on 1.2.0 version, but I can't use it on 1.3.0 version.
So I'm not able to use T3 with Browserify (for example).

Maybe a problem with the release script ?
It should include this file : [lib/wrap-end.partial](/box/t3js/blob/master/lib/wrap-end.partial)
But it doesn't seem to include it...

Thanks !

### PR DESCRIPTION
**Pull request by j3tan**
CommonJS and AMD support was accidentally removed for 1.3.0

fixes #72

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @j3tan has signed the CLA. Thanks for the pull request!

**Comment by nzakas** [comment]
Change looks good. Travis is failing due to this: https://github.com/cjohansen/Sinon.JS/issues/761

Also, Is there a way to add a test to ensure CommonJS support isn't broken?

**Comment by j3tan** [comment]
Test would have to run `npm run dist` which might not be great. Have you seen examples of testing AMD/commonJS wrappers like this?

**Comment by nzakas** [comment]
I'd just run during a release after `dist`. All you really need to do is `require('./dist/t3.js')` and see that it returns the correct object.

File: Makefile.js [hunk 1]
```diff
@@ -273,21 +290,21 @@ target.dist = function() {
...
-        files: SRC_JQUERY_FILES,
-        testingFiles: TESTING_JQUERY_FILES
-    }, {
-        name: DIST_NAME,
-        files: SRC_JQUERY_FILES,
-        testingFiles: TESTING_JQUERY_FILES
-    }].forEach(function(dist){
-        generateDistFiles(dist);
-    });
+	[{
```
Comment (j3tan) [inline Makefile.js]: just tab changes

**Comment by nzakas** [comment]
+2

Sinon was also fixed

**Pull request merged**